### PR TITLE
[Backend][Issue #103] Help EndPoint

### PR DESCRIPTION
**Pull request by JossEsparza**
Hi @vinfante  @ErickRodriguezWize

Here is the code for Issue https://github.com/wizeline/ror-remotepayment-wizeline/issues/103

Closes https://github.com/wizeline/ror-remotepayment-wizeline/issues/103

### CODE REVIEW

File: remote_payment_back/app/controllers/api/v1/application_controller.rb [hunk 1]
```diff
@@ -121,7 +121,7 @@ def user_is_contractor?
 
   # Returns if the logged user is a Finance staff
   def user_is_finance?
-    user_by(@token)&.role&.name == Role::FINANCE_ROLE
+    [Role::FINANCE_ROLE, Role::FINCANCE_HELP_ROLE].include?(user_by(@token)&.role&.name)
```
Comment (vinfante) [inline remote_payment_back/app/controllers/api/v1/application_controller.rb:124]: Fix typo, FINCANCE to FINANCE

**Review by vinfante — CHANGES_REQUESTED** [review]
All ok, just fix functional typos

File: remote_payment_back/app/models/role.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@ class Role < ApplicationRecord
 
   CONTRACTOR_ROLE = 'Contractor'.freeze
   FINANCE_ROLE = 'Finance Staff'.freeze
+  FINCANCE_HELP_ROLE = 'Finance Help'.freeze
```
[the remark this answers is not in the document]
Comment (JossEsparza) [reply remote_payment_back/app/models/role.rb]: Fixed with Commit:
a62cdb8091839080a5c985f931dc330c71ce90f4 [a62cdb8]

File: remote_payment_back/app/controllers/api/v1/application_controller.rb (hunk 1 above)
Comment (JossEsparza) [reply remote_payment_back/app/controllers/api/v1/application_controller.rb]: Fixed with Commit:
a62cdb8091839080a5c985f931dc330c71ce90f4 [a62cdb8]

File: remote_payment_back/app/models/user.rb [hunk 3]
```diff
@@ -17,6 +17,11 @@ class User < ApplicationRecord
                                 .where('roles.name = ?', Role::CONTRACTOR_ROLE).order(name: :asc, surname: :asc)
                             }
 
+  scope :finance_help_sorted, lambda {
+                                joins(:role)
+                                  .where('roles.name = ?', Role::FINCANCE_HELP_ROLE).order(name: :asc, surname: :asc)
```
[the remark this answers is not in the document]
Comment (JossEsparza) [reply remote_payment_back/app/models/user.rb]: Fixed with Commit:
a62cdb8091839080a5c985f931dc330c71ce90f4 [a62cdb8]

**Review by vinfante — APPROVED** [review]
LGTM
again

**Pull request merged**